src/components/mainContent/ForwardButton.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from "react";
+import "./mainContent.css";
+import arrows from "../../assets/upArrow.svg";
+
+export default function ForwardButton(props) {
+  let currentSection = props.content[props.visibleSections];
+  let currentLength = undefined;
+  if (currentSection && currentSection.post)
+    currentLength = currentSection.post.bodyText.length;
+  else if (currentSection && currentSection.followups)
+    currentLength = currentSection.followups.length;
```

Comment:
This looks too similar to the code in `DirectionButtons.js`, try to de-duplicate it if you can

Reply:
Yup, deleting DirectionButtons, just forgot to delete our previous implementation of it